### PR TITLE
Donate link from Rocket To Open Collective

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Netlify hosted:
 [//]: # (numfocus-fiscal-sponsor-attribution)
 
 The Insight Software Consortium (ISC) uses an [open governance model](./GOVERNANCE.md)
-and is fiscally sponsored by [NumFOCUS](https://numfocus.org/). Consider making a [tax-deductible donation](https://numfocus.org/donate-to-itk)
+and is fiscally sponsored by [NumFOCUS](https://numfocus.org/). Consider making a [tax-deductible donation]()https://opencollective.com/itk
```
Comment (jhlegarreta) [inline README.md:16]: Syntax error here for the closing bracket.

I guess this change should also be made to ITK, ITKExamples, ITK SW Guide, SimpleITK.

**Review by jhlegarreta — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (thewtex) [reply README.md:16]: :+1: 

I will also make PR's to the other repositories.

**Review by jhlegarreta — APPROVED** [review]
Thanks Matt. If you have the time, as the changes/commits are wildly different, maybe the PR subject could reflect both, and the commit message subjects be copied to the PR message body.

File: README.md (hunk 1 above)
Comment (thewtex) [reply README.md]: I made a PR against ITK. https://github.com/InsightSoftwareConsortium/ITK

I did not find links on ITKSphinxExamples, SimpleITK, ITKSoftwareGuide.

Comment (jhlegarreta) [reply README.md]: OK, maybe we discussed this at some point and preferred not to add it. I cannot remember. Thanks Matt.

Resolving.

**Pull request merged**